sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -114,6 +116,18 @@ class JsonFunctionsSuite extends QueryTest with SharedSQLContext {
       Row(Row(null)) :: Nil)
   }
 
+  test("from_json null column with parquet source") {
+    withTempDir { dir =>
+      Seq("""{"a": 1}""").toDF("value").write.parquet(new File(dir, "p=x").toString)
+      Seq("""{"a": 2}""").toDF("record").write.parquet(new File(dir, "p=y").toString)
+      val schema = new StructType().add("a", IntegerType)
+      val df = spark.read.parquet(dir.toString)
```

Comment:
Why parquet?  Seems this is testing more than it needs to (and thus runs slower).